apps/dsp-app/src/app/project/ontology-classes/ontology-class-item/ontology-class-item.component.ts

```diff
@@ -111,6 +115,20 @@ export class OntologyClassItemComponent implements OnInit, AfterViewInit, OnDest
     }
   }
 
+  /**
+   * Gets the ontology label in the preferred language of the user
+   */
+  private getOntologiesLabelsInPreferredLanguage(): void {
+    // TODO: this method and changes in below PR are workourund, the label selection should be invastigated deeper to find out why DE is set as default
+    // https://github.com/dasch-swiss/dsp-das/pull/1651
+    const prefferedLanguage = this._store.selectSnapshot(UserSelectors.language);
+
+    if (this.resClass.labels) {
```

Comment:
so if this condition is not met, ontologiesLabel stays undefined, is that what we want ? (just asking I don't know what's the right behaviour)

Reply:
I think this is not necessary, because of the guard I added: `this.resClass.labels[0].value`. If there is no match between label languages and preferred language, the first one from the labels array is applied, and at least one label is required AFAIK.